script.service.ltvpl/resources/skins/Default/720p/contextmenu.xml

```diff
@@ -0,0 +1,146 @@
+<window>
+  <coordinates>
+    <system>1</system>
+    <posx>470</posx>
+    <posy>255</posy>
+  </coordinates>
+  <!--PreviousMenu>Home</PreviousMenu-->
+  <!--visible>!Window.IsActive(Window.Property(parentWinID))</visible-->
+  <controls>
+    <!--item background image 1-->
+    <control type="image" id="5001">
+      <posx>0</posx>
+      <posy>0</posy>
+      <width>342</width>
+      <height>96</height>
+      <texture border="20">DialogBack.png</texture>
+    </control>
+    <!--item background image 2-->
+    <control type="image" id="5002">
+      <posx>0</posx>
+      <posy>0</posy>
+      <width>342</width>
+      <height>134</height>
+      <texture border="20">DialogBack.png</texture>
+    </control>
+    <!--item background image 3-->
+    <control type="image" id="5003">
+      <posx>0</posx>
+      <posy>0</posy>
+      <width>342</width>
+      <height>172</height>
+      <texture border="20">DialogBack.png</texture>
+    </control>
+    <!--item background image 4-->
+    <control type="image" id="5004">
+      <posx>0</posx>
+      <posy>0</posy>
+      <width>342</width>
+      <height>210</height>
+      <texture border="20">DialogBack.png</texture>
+    </control>
+    <!--List to hold 'buttons'-->
+    <control type="list" id="3000">
+      <posx>20</posx>
+      <posy>30</posy>
+      <width>300</width>
+      <height>152</height>
+      <itemwidth>300</itemwidth>
+      <itemheight>38</itemheight>
+      <pagecontrol>4000</pagecontrol>
+      <scrolltime>200</scrolltime>
+      <!--
+        jmoore Mod
+        Purpose: Fix Navigation to accomodate invisible scrollbar
+      -->
+      <onright condition="Control.IsVisible(4000)">4000</onright>
+      <onright condition="!Control.IsVisible(4000)">3001</onright>
+      <onleft condition="Control.IsVisible(4000)">4000</onleft>
+      <onleft condition="!Control.IsVisible(4000)">3001</onleft>
+
+      <itemlayout width="300" height="38">
+        <control type="image">
+          <posx>0</posx>
+          <posy>0</posy>
+          <width>300</width>
+          <height>38</height>
+          <texture border="5">button-nofocus.png</texture>
+        </control>
+        <control type="label">
+          <posx>50</posx>
+          <posy>0</posy>
+          <width>250</width>
+          <height>38</height>
+          <font>fontContextMenu</font>
```

Comment:
there's no guaranty `fontContextMenu` is defined by the skin you're using

Reply:
removed